slave.mk

```diff
@@ -2,7 +2,7 @@
 ## Presettings
 ###############################################################################
 
-# Select bash for commands
+# Select bash for commands 
```

Comment:
just to trigger changes ?

Reply:
Yes. I had issue with VS tests failing and this helps.